galeracluster/source/monitoringthecluster.rst

```diff
@@ -1,305 +1,288 @@
-   
-   Check the error log for proper diagnostics.
-
-3. If the node is connected, but still cannot accept SQL load, check that the node is part of the Primary Component:
-
-   .. code-block:: mysql
-
-	SHOW GLOBAL STATUS LIKE 'wsrep_local_state_comment';
-   
-   If the the state comment is ``Joining``, ``Waiting for SST``, or ``Joined``, the node is syncing with the cluster.  
-   
-   In a Primary Component, the state comment typically returns ``Joining``, ``Waiting for SST``, ``Joined``, ``Synced``, or ``Donor``.  In other components, the state comment returns ``Initialized``.  Any other state comments are transient and momentary.
-
-Should each of these status variables check out, then the node is in working order.  It is receiving write-sets from the cluster and replicating them to tables on the local database.
-
-
-
----------------------------------
- Checking the Replication Health
----------------------------------
-.. _`check-replication-health`:
-
-.. index::
-   pair: Parameters; wsrep_flow_control_paused
-
-.. index::
-   pair: Parameters; wsrep_cert_deps_distance
-
-.. index::
-   pair: Parameters; wsrep_local_recv_queue_avg
-
-.. index::
+===============================
+ Monitoring Cluster Status
+===============================
+.. _`monitoring-cluster`:
+
+.. index::
+   pair: Parameters; wsrep_notify_cmd
+
+From the database client, you can check the status of write-set replication throughout the cluster using standard queries.  Status variables that relate to write-set replication have the prefix ``wsrep_``, meaning that you can display them all using the following query:
+
+.. code-block:: mysql
+
+   SHOW GLOBAL STATUS LIKE 'wsrep_%';
+
+   +------------------------+-------+
+   | Variable_name          | Value |
+   +------------------------+-------+
+   | wsrep_protocol_version | 5     |
+   | wsrep_last_committed   | 202   |
+   | ...                    | ...   |
+   | wsrep_thread_count     | 2     |
+   +------------------------+-------+
+
+.. seealso:: In addition to checking status variables through the database client, you can also monitor for changes in cluster membership and node status through ``wsrep_notify_cmd.sh``.  For more information on its use, see :doc:`notificationcmd`.
+
+
+---------------------------------
+ Checking Cluster Integrity
+---------------------------------
+.. _`check-cluster-integrity`:
+
+.. index::
+   pair: Parameters; wsrep_cluster_state_uuid
+.. index::
+   pair: Parameters; wsrep_cluster_conf_id
+.. index::
+   pair: Parameters; wsrep_cluster_size
+.. index::
+   pair: Parameters; wsrep_cluster_status
+
+The cluster has integrity when all nodes in it receive and replicate write-sets from all other nodes.  The cluster begins to lose integrity when this breaks down, such as the cluster is down, become partitioned, or experiences a split-brain situation.
```

Comment:
"such as the cluster is down, become partitioned"

such as when the cluster goes down, becomes partitioned or experiences ...